### PR TITLE
Fix broken --filename option for 'oc env' cmd

### PR DESCRIPTION
**Pull request by pravisankar**
cleanup: remove unused method

### CODE REVIEW

**Comment by pravisankar** [comment]
@smarterclayton @danmcp minor change, tested on my local machine. Can we get this into master?

File: pkg/cmd/cli/cmd/env.go [hunk 1]
```diff
@@ -64,7 +63,7 @@ func NewCmdEnv(fullName string, f *clientcmd.Factory, in io.Reader, out io.Write
 	var filenames util.StringList
 	var env util.StringList
 	cmd := &cobra.Command{
-		Use:     "env RESOURCE/NAME KEY_1=VAL_1 ... KEY_N=VAL_N [options]",
+		Use:     "env RESOURCE/NAME KEY_1=VAL_1 ... KEY_N=VAL_N",
```
Comment (smarterclayton) [inline pkg/cmd/cli/cmd/env.go]: Why did you remove [options]?  Does cobra add it already?

Comment (pravisankar) [inline pkg/cmd/cli/cmd/env.go]: yes

**Comment by smarterclayton** [comment]
LGTM [merge]
